src/backEnd/server.js

```diff
@@ -0,0 +1,19 @@
+const express = require('express');
+const path = require('path');
+const bodyParser = require('body-parser');
+
+const router = require('./router');
+
+const app = express();
+console.log(path.join(__dirname, "../.."));
```

Comment:
a whole lots of console logs in this file that were used for debugging so they are not needed  ow :sweat_smile: